def raise_exception():
    1 / 0
